healthyregions/chicago-environment-explorer
# feat: update Observed PM2.5 sensor data

## Problem
Observed Particulate Matter (PM2.5) data needs to be updated

Fixes #148 

## Approach
* feat: use new data + metadata links from @jluiuc to display this data in the ChiVes frontend
* fix: update "Source Data" CSV file to use more recent readings
* fix: restore/rename overlay to "Microsoft PM2.5 Readings" - these are historical sensor readings that may or may not match current color values on the map
* fix: use new color scheme (YlOrBr w/ 8 bins) for all Air Quality indicators + Source Data toggle

![Screenshot 2024-07-29 at 10 49 09 AM](https://github.com/user-attachments/assets/9543abda-f1c2-443e-8597-58a5868cf3ce)

## How to Test
1. Navigate to https://deploy-preview-152--chicago-env-explorer.netlify.app/map
2. From the Variables dropdown, select "Observed Particulate Matter (PM2.5)" from the Air Quality category
    * You should see the data is displayed in the map to the right
    * You should see some overlayed sensor readings appear in the map (do these values need to be updated as well?)
    * You should see some additional controls in the panel, such as "2D or 3D" and "Aggregated by Tract or Source Data"
3. Toggle from "Aggregated by Tract" to "Source Data"
    * You should see that different values now appear in the map
4. Toggle from 2D to 3D
    * You should see the perspective shift slightly to show a 3D rendering of this data on the map
